### DISCUSSION TITLE
Main window like Mac OS X app ?

### DISCUSSION THREAD

**Discussion by guldud** [kikipoulet/SukiUI#269] [General]
Hi,

Love the concept and your approach.

The apps using Suki have a great MS Windows look. Is the Mac OS X window look on the "roadmap" ?  I'm trying to hit the look of the Mac OS X Weather App.

**Comment by kikipoulet** [comment]
Hi, thank for your kind words.

This is quite an unexpected comment. I like it because it shows how perception of design can be so different between peoples. To me, the library is far from Windows and closer from clean macos design, even if I think it has is own identity obviously (I hope so).

![356755017-23b03fd1-4ee0-4554-9704-8bbfc5bae27a.png](https://github.com/user-attachments/assets/f0f903ed-16fc-4cb4-861a-9e668c8f3cd4)


So I'm quite curious to know why you consider it closer to Windows actually, and how a macos version would be 🤔

**Reply by guldud** [reply]
I understand. 

To me the chrome bar is a clear (and dear) indication of a Mac OS X app. I also really like the way the sidebar icon "moves" (it does not).  You can see this in weather app, the shortcuts app, and many other apps.

<img width="1105" alt=" weather sidebar hidden" src="https://github.com/user-attachments/assets/a2cd3d40-8f80-459f-8956-9f106a186c8b">
<img width="1105" alt="weather sidebar shown" src="https://github.com/user-attachments/assets/4b56c467-1792-462d-86fa-b7d897b1480a">

**Comment by kikipoulet** [comment]
Old discussion, but managed to find a way to do that properly creating a "ShowTitleBarBackground" property. The problem was what to do with Menu and SideMenu in this situation, design wise.



https://github.com/user-attachments/assets/9a32f9cd-a348-404e-94eb-201bd609b5bd


Using Menu and SideMenu with no TitleBarBackground makes not a lot of sense, but it's important that they can adapt if needed. An office-like application (#369) could benefit from using the Menu as a ribbon in this situation.
